src/app/modules/layout/components/sidebar/sidebar.component.html

```diff
@@ -0,0 +1 @@
+<p>sidebar works!</p>
```

Comment:
As I see, it's an empty component, so why do you need it?

Reply:
I`ll make it after this pull request.